### PR TITLE
feat: add json output

### PR DESCRIPTION
**Pull request by stephhuynh18**

### CODE REVIEW

File: src/benchie/bin/benchie.ts [hunk 1]
```diff
@@ -44,13 +44,16 @@ async function main() {
 
   const runner = new Runner(dir);
   const results = await runner.run();
+  const currentDir = new URL(
+    `file://${path.resolve(new URL(".", cwd).pathname)}/`
```
Comment (ukstv) [inline src/benchie/bin/benchie.ts]: @stephhuynh18 why `path.resolve` is needed here? Why `new URL(".", cwd)` is not enough for `currentDir`?

File: src/benchie/results-output.ts [hunk 2]
```diff
@@ -23,3 +30,25 @@ export class ResultsOutputText {
...
+  constructor(readonly results: IScenarioResult[], readonly dir: URL) {}
+
+  print() {
+    const grouped = groupResults(this.results);
+    const output = {};
+    for (let [href, results] of grouped.entries()) {
+      const filepath = new URL(href);
+      const filename = filepath.href.replace(this.dir.href, "");
+      for (let r of results) {
+        output[filename] = {
```
Comment (ukstv) [inline src/benchie/results-output.ts]: what if there are two scenarios in one file?

File: src/benchie/bin/benchie.ts (hunk 1 above)
Comment (ukstv) [reply src/benchie/bin/benchie.ts]: Genuine question. I am confused. I feel, like there is some magic that I miss here.

File: src/benchie/results-output.ts (hunk 2 above)
Comment (ukstv) [reply src/benchie/results-output.ts]: IMO, only the last one would be reported.

File: src/benchie/bin/benchie.ts (hunk 1 above)
Comment (stephhuynh18) [reply src/benchie/bin/benchie.ts]: I actually just copied this line haha: 
```
  const dir = new URL(
    `file://${path.resolve(new URL(options.dir, cwd).pathname)}/`
  );
  ```
  https://github.com/ceramicnetwork/benchie/blob/102d7ab6969111f384c3b56076bef5b431bcd437/src/benchie/bin/benchie.ts#L41
  
But yes you are right i do not need path.resolve

File: src/benchie/results-output.ts (hunk 2 above)
Comment (stephhuynh18) [reply src/benchie/results-output.ts]: yes didn't account for multiple scenarios in one file. Will change that. I also want nested scenarios but that will be another time

**Pull request merged**